sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -30,33 +30,28 @@ import org.apache.spark.sql.test.{ExamplePointUDT, ExamplePoint, SQLTestUtils}
 class DataFrameSuite extends QueryTest with SQLTestUtils {
   import org.apache.spark.sql.TestData._
 
-  lazy val ctx = org.apache.spark.sql.test.TestSQLContext
-  import ctx.implicits._
-
-  def sqlContext: SQLContext = ctx
+  val sqlContext = org.apache.spark.sql.test.TestSQLContext
```

Comment:
lazy val here - otherwise a failure in the ctor could wipe out the entire test suite and then test cases won't show up as a failed tests.